portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,43 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+var slideIndex = 1;
+showSlides(slideIndex);
+
+function plusSlides(offset) {
+  showSlides(slideIndex += offset);
+}
+
+function currentSlide(newSlideIndex) {
+  showSlides(slideIndex = newSlideIndex);
+}
+
+function showSlides(activatedSlide) {
```

Comment:
As a stretch goal (not requiring action here), you could ask yourself how you would write a unit test e.g. to make sure that the wrap-around / indexing logic is all working.

Reply:
I believe I would write some input tests and use an "expected output" reference. Then I will just compare the function's output with the expected output.